vignettes/using-cross-table.Rmd

```diff
@@ -39,12 +38,12 @@ Inside this app 5 datasets will be used
 5. `ADLB` A long data set with lab measurements for each subject
 
 ```{r echo=TRUE, message=FALSE, warning=FALSE, results="hide"}
-ADSL <- synthetic_cdisc_data("latest")$adsl # nolint
-ADSL2 <- synthetic_cdisc_data("latest")$adsl %>% # nolint
+ADSL <- teal.modules.general::rADSL # nolint
+ADSL2 <- teal.modules.general::rADSL %>% # nolint
   mutate(TRTDUR = round(as.numeric(TRTEDTM - TRTSDTM), 1))
-ADRS <- synthetic_cdisc_data("latest")$adrs # nolint
-ADTTE <- synthetic_cdisc_data("latest")$adtte # nolint
-ADLB <- synthetic_cdisc_data("latest")$adlb %>% # nolint
+ADRS <- teal.modules.general::rADRS # nolint
+ADTTE <- teal.modules.general::rADTTE # nolint
+ADLB <- teal.modules.general::rADLB %>% # nolint
```

Comment:
@kartikeyakirar is there a way to assign the whole chunk to be non-linted, so that we do not need to have nolint in every line?

Reply:
checkout this commit a0d16f39d30a01041e0ce661a1d8af2ecd70c5e0 which removes this weird #nolint in every line of vignette output
![image](https://github.com/insightsengineering/teal.modules.general/assets/133694481/f3d99e2d-d95f-4906-8896-86d7cfdfee28)

this is based on on this thread https://github.com/r-lib/lintr/issues/773

Reply:
this awesome!